### DISCUSSION TITLE
Errors MSB3027 and MSB3021 occurred when building the project.

### DISCUSSION THREAD

**Discussion by xiejiang2014** [AvaloniaUI/Avalonia#11909] [General]
In VS2022, as soon as I open any of the .axaml files, the build fails.
There will be two errors:
MSB3027: Could not copy 'source' to 'dest'. Exceeded retry count of 'number'. Failed, File was locked by .Net Host
MSB3021: Unable to copy file 'source' to 'destination'. The file '***' is being used by another process

I need to close all .axaml files in order to build the project successfully.

Maybe disabling the Avalonia previewer would solve the problem, but I didn't find a switch to disable it.

**Recently found that this problem only occurs when "ReSharper Build" is activated.**

**Comment by timunie** [comment]
I feel like this is a bug introduced with a recent VS update as I've heard from a friend having the same error while working on a larger WPF app. 

However, if you think it's the previewer then try to disable the entire Avlaonia plugin an test the project again.

**Reply by xiejiang2014** [reply]
Thanks.
I think this may not be a recent problem, because I also encountered the same problem when using vs2019 a few years ago.
Yes, disabling the Avlaonia plugin can avoid this problem, but at the same time i will lose other functions provided by the plugin.

**Comment by xiejiang2014** [comment]
Surprisingly, when I disabled ReSharper, the problem disappeared.

**Reply by timunie** [reply]
That's interesting. I'll tell my friend. Afaik he also uses ReSharper ^^
